static/js/zamboni/devhub.js

```diff
@@ -335,11 +335,13 @@ function addonFormSubmit() {
             parent_div.find(".item").removeClass("loaded").addClass("loading");
             var $document = $(document),
                 scrollBottom = $document.height() - $document.scrollTop(),
-                $form = $(this),
-                hasErrors = $form.find('.errorlist').length;
+                $form = $(this);
 
             $.post($form.attr('action'), $form.serialize(), function(d) {
+                // The HTML has changed after we posted the form, thus the need to retrieve the new HTML
```

Comment:
Can you move that comment one line down ? Since that's the line below that changes the HTML.

Reply:
Done.